roxctl/central/crs/generate.go

```diff
@@ -55,11 +55,11 @@ func generateCRS(cliEnvironment environment.Environment, name string,
 	}
 
 	var resp *v1.CRSGenResponse
-	if validFor != 0 || !validUntil.IsZero() {
-		resp, err = generateCrsExtended(ctx, svc, name, validFor, validUntil)
+	if validFor != 0 || !validUntil.IsZero() || maxClusters != 0 {
+		resp, err = generateCrsExtended(ctx, svc, name, validFor, validUntil, maxClusters)
 		if err != nil {
 			if errStatus, ok := status.FromError(err); ok && errStatus.Code() == codes.Unimplemented {
-				cliEnvironment.Logger().ErrfLn("generating a CRS with custom expiration times requires a newer Central")
+				cliEnvironment.Logger().ErrfLn("generating a CRS with extended expiration settings requires a newer Central")
```

Comment:
I would suggest being more precise here:
- Whenever we write `newer` there should be always a `than` part - maybe we can add it?
- Alternatively, we can rephrase into:


```suggestion
				cliEnvironment.Logger().ErrfLn("generating a CRS with extended expiration settings is not supported in the current version of Central")
```